docs/source/logging/index.md

```diff
@@ -42,6 +42,17 @@ After setting the environment variable, any subsequent Kedro commands use the lo
 If the `KEDRO_LOGGING_CONFIG` environment variable is not set, Kedro will use the [default logging configuration](https://github.com/kedro-org/kedro/blob/main/kedro/framework/project/default_logging.yml).
 ```
 
+One can also customise logging at the runtime and redefine the logging configuration provided in the `logging.yml`.
```

Comment:
```suggestion

### Change the verbosity of specific parts of Kedro

One can also customise logging at the runtime and redefine the logging configuration provided in the `logging.yml`.
```

Reply:
"You can" instead of "One can", maybe? Most of the docs are in second person I think